File: client/src/main/java/com/gaganbelgur/BuilderClient.java
package com.gaganbelgur;

public class BuilderClient {

    public static void main(String[] args) {

        ProfileJava profileJava = new ProfileJava.ProfileBuilder("GaganBelgur", "9876543210").build();
        System.out.println(profileJava.toString());

        ProfileJava profileJava1 = new ProfileJava.ProfileBuilder("GaganBelgur", "9876543210").setBirthDate("25/10/20").build();
        System.out.println(profileJava1.toString());
    }
}

File: client/src/main/kotlin/com/gaganbelgur/BuilderClient.kt
package com.gaganbelgur

import com.gaganbelgur.com.gaganbelgur.Profile

fun main() {

    val profile = Profile.ProfileBuilder().setName("GaganBelgur").build()
    println(profile.name)

    val profile1 = Profile.ProfileBuilder()
        .setName("GaganBelgur")
        .setEmail("email")
        .build()

    println(profile1)
}
